src/pixelator/analysis/colocalization/__init__.py

```diff
@@ -341,20 +340,13 @@ def _colocalization_wilcoxon(
         alternative="two-sided",
     )
 
-    return {
-        "marker_1": markers[0],
-        "marker_2": markers[1],
-        "markers": "/".join(markers),
-        "stat": stat,
-        "p_value": p_value,
-        "median_difference": estimate,
-    }
+    return pd.Series({"stat": stat, "p_value": p_value, "median_difference": estimate})
 
 
 def get_differential_colocalization(
     colocalization_data_frame: pd.DataFrame,
-    target: str,
     reference: str,
+    targets: str | list[str] | None = None,
     contrast_column: str = "sample",
     use_z_score: bool = True,
```

Comment:
If we add a different score in the future, e.g. conditional probability or join-counts, would it make sense to enable setting this manually with a `value_column ` argument?

Reply:
Good point, we can change the input to be value_column directly since we are already changing function inputs anyway.